test-coverage.txt

```diff
@@ -2,52 +2,54 @@
 > react-certification-2020@0.1.0 coverage /Users/aaron/Desktop/react-certification-2020
 > react-scripts test --env=jsdom --watchAll=false --coverage
 
- PASS  src/components/LoginButton/Login.test.js (5.032s)
- PASS  src/components/Card/Card.test.js (5.994s)
- PASS  src/index.test.js
- PASS  src/_test_/getYoutubeData.test.js (8.039s)
+ PASS  src/components/Toogle/ToogleTheme.test.js
+ PASS  src/pages/VideoDetail/VideoDetail.test.js
+ PASS  src/_test_/getYoutubeData.test.js (6.167s)
+ PASS  src/components/Carrusel/Carrusel.test.js (7.127s)
  PASS  src/components/App/App.test.js
  PASS  src/components/Layout/Layout.test.js
- PASS  src/components/Toogle/ToogleTheme.test.js
+ PASS  src/components/LoginButton/Login.test.js
+ PASS  src/components/Card/Card.test.js
+ PASS  src/index.test.js
  PASS  src/components/Menu/Menu.test.js
  PASS  src/components/Header/Header.test.js
  PASS  src/pages/NotFound/NotFound.test.js
  PASS  src/components/InputSearch/InputSearch.test.js
 ----------------------------|----------|----------|----------|----------|-------------------|
 File                        |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
 ----------------------------|----------|----------|----------|----------|-------------------|
-All files                   |    68.99 |    14.29 |       55 |    69.05 |                   |
+All files                   |    79.73 |    19.05 |    73.33 |    79.86 |                   |
```

Comment:
Code coverage looks good, most of the time the branch percentage is the trickiest one